FirebaseAppCheck.podspec

```diff
@@ -43,7 +43,7 @@ Pod::Spec.new do |s|
   s.tvos.weak_framework = 'DeviceCheck'
 
   s.dependency 'FirebaseCore', '~> 8.0'
-  s.dependency 'PromisesObjC', '>= 1.2', '< 3.0'
```

Comment:
Should be 2.0 and not 2.0.0 to allow minor updates as well as patch updates. Also the '< 3.0' should be removed. Details about syntax at https://guides.cocoapods.org/syntax/podspec.html#dependency

Reply:
Fixed